src/mlpack/methods/fastmks/fastmks.hpp

```diff
@@ -141,6 +184,29 @@ class FastMKS
    * @param tree Tree to use as reference data.
```

Comment:
To mark a method deprecated, also add `mlpack_deprecated` to the function definition please.  i.e. `mlpack_deprecated void Train(...)`.